src/transport/PointToPointBroker.cpp

```diff
@@ -611,6 +746,13 @@ void PointToPointBroker::resetThreadLocalCache()
     clients.clear();
 }
 
+bool PointToPointBroker::setIsMessageOrderingOn(bool mustOrderMsgs)
+{
+    bool oldValue = _isMessageOrderingOn.load(std::memory_order_acquire);
+    _isMessageOrderingOn.store(mustOrderMsgs, std::memory_order_release);
+    return oldValue;
```

Comment:
Can we use `exchange` here rather than `load` and `store`?

Reply:
I was not aware of `exchange`, thanks.